android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/service/address/WalletAddressService.java

```diff
@@ -0,0 +1,70 @@
+package com.appcoins.sdk.billing.service.address;
+
+import com.appcoins.billing.sdk.BuildConfig;
+import com.appcoins.sdk.billing.mappers.AddressResponseMapper;
+import com.appcoins.sdk.billing.models.AddressModel;
+import com.appcoins.sdk.billing.service.RequestResponse;
+import com.appcoins.sdk.billing.service.Service;
+import com.appcoins.sdk.billing.service.ServiceResponseListener;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.appcoins.sdk.billing.payasguest.AdyenPaymentInteract.AddressListener;
+
+public class WalletAddressService {
+
+  private final String defaultStoreAddress;
+  private final String defaultOemAddress;
+  private Service service;
+
+  public WalletAddressService(Service service) {
+    this.service = service;
+    defaultStoreAddress = BuildConfig.DEFAULT_STORE_ADDRESS;
```

Comment:
why don't you receive these values as parameters?

Reply:
Agree